### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by SimonasVsm**
Updated final version for today.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,179 @@
+:root {
+  --primary: #06c;
+  --secondary: rgb(137, 170, 202);
+  --success: rgb(24, 201, 24);
+  --failure: rgb(192, 59, 59);
```
Comment (visma-zanas-stundys) [inline styles.css]: Consider using `hsl` format, could be more readable ;)

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+.nav {
+  display: flex;
+  justify-content: space-between;
+  background-color: var(--secondary);
+  padding: 1rem;
+}
+
+.nav__logo {
+  /* flex: 2; */
+}
```
Comment (visma-zanas-stundys) [inline styles.css]: Avoid commiting unused code :)

**Review by visma-zanas-stundys — CHANGES_REQUESTED** [review]
Nice :)

**Review by LukasTyla — COMMENTED** [review]
Looks great, it's nice to see css variables used.

**Pull request merged**